### ISSUE TITLE
scroll

### ISSUE THREAD

**Issue by szn0212** [lawloretienne/QuickReturn#23]
When I scroll listview,the footer is flashing

**Comment by rabyunghwa** [comment]
I have the same issue. And if i scroll very slowly, the QuickReturnFooterTextView and the QuickReturnFooterImageView(Google plus implementation) do not scroll at the same pace.

**Comment by lawloretienne** [comment]
That google plus implementation is by design. That is how it was being
handled in the Google + app. Keep in mind it is just an example. You are
free to customize that behavior.

**Comment by rabyunghwa** [comment]
@lawloretienne ok man. thanks.

**Comment by zhonghanwen** [comment]
@rabyunghwa have a solution?

**Comment by rabyunghwa** [comment]
@zhonghanwen nope. I decided to use the header textview instead and just manually set its visibility to visible or gone. This is what the current behavior is in the latest Facebook's app.
